.travis.yml

```diff
@@ -1,10 +1,14 @@
 language: go
+env: GO111MODULE=on
 go:
-  - 1.11
-  - 1.14.2
-before_install:
-  - go get -t -v ./...
+  - 1.11.x
```

Comment:
can we explicitly write 1.11.4, I believe .x will pull the latest version which is `1.11.13`.

Reply:
both works.

Reply:
use 1.11.4 @Piyushhbhutoria @vaibhavsethia